.github/workflows/build-centraldashboard.yml

```diff
@@ -6,6 +6,8 @@ on:
     paths:
       - components/centraldashboard/**
   pull_request:
+    paths:
+      - components/centraldashboard/**
```

Comment:
don't want to build `CentralDashboard` when contributing to other parts of the repo.

Reply:
makes sense.